localstack/services/cloudformation/resource_provider.py

```diff
@@ -239,6 +239,7 @@ def invoke_function(
                 raise
 
             LOG.debug("Converting parameters to allowed types")
+            LOG.debug(f"Report: {report}")
```

Comment:
```suggestion
            LOG.debug("Report: %s", report)
```
To avoid unnecessary string interpolation if debug isn't enabled.

Reply:
Ahh good call. I'll add that.